GiEnJul/ClientApp/src/components/admin/common/Datatable.tsx

```diff
@@ -59,15 +59,13 @@ const Datatable: React.FC<Props> = ({ data }) => {
               <Group />
               {formatFamily(x.maxRecievers)}
```

Comment:
Typo in maxRecievers. This should be maxReceivers instead. This is an issue other places as well. See issue #202. Could this be changed in this task, or should it be changed in issue #202?

Reply:
Is it a spelling error? If so, I think we can change this quickly by replace all with the correct spelling.